### ISSUE TITLE
A11y - Login Page

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1384]
## User story

Prio 3:

- [ ] ~~Fix heading hyerarchy~~
- [x] Fix contrast issues
- [ ] ~~Fix landmark usage (empty or contentless sections)~~
- [x] Make input focus state more prominent (reusable design token)
- [ ] ~~Checkbox does not have visible focus state, make it visible, prominent and consistent with rest of site~~
- [x] Form error state is not a11y: errors not announced because they are not a live region or describe the inputs, also, there's no clear visual cue that it is an error. Reference: https://webaim.org/techniques/formvalidation/
  - [x] screen readers announce errors when they appear
  - [ ] ~~errors are visually distinct for any user (do not depend only on color, use error icon and perhaps error label)~~
- [x] form required labels not a11y (i.e. `aria-required="true"` or `required` attribute)

Prio 2

- [x] Cient side validation: judging from the UX, form is submitted and then server side validation happens. Client side validation would improve the UX, ref https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation

Prio 1

- [ ] ~~Broken layout: footer is above viewport bottom~~ this happens everywhere so leave fix for another ticket

https://beta.gigadb.org/site/login

**Comment by luistoptal** [comment]
- Color contrast fixed in #1363
- Landmark usage fixed in #1361
- Checkbox does not have visible focus state: fixed in #1371
